src/components/loading/loading.js

```diff
@@ -0,0 +1,18 @@
+import { css } from '@emotion/react';
+import PuffLoader from 'react-spinners/PuffLoader';
+
+// Can be a string as well. Need to ensure each key-value pair ends with ;
+const override = css`
```

Comment:
interesting. is `@emotion/react` needed to override the css for `PuffLoader`? (is it the only option?)

Reply:
according to dependence, it is a requirement to use, I'll be researching it!